classes/classes/Items/Consumable.as

```diff
@@ -16,7 +16,6 @@ package classes.Items
 		include "../../../includes/appearanceDefs.as";
 
 		protected function get mutations():Mutations { return kGAMECLASS.mutations; }
-		protected function set mutations(val:Mutations):void { kGAMECLASS.mutations = val; }
```

Comment:
The setter was unused, useless and stupid. Another reason for turning Mutations into a Singleton.